RNASeqPipeline.Rmd

```diff
@@ -6,139 +6,129 @@ output: pdf_document
 
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
-```
-
-```{r}
-install.packages("BiocManager")
-BiocManager::install("DESeq2")
-```
 
-```{r}
+# Load libraries
 library(DESeq2)
 library(onlineFDR)
 library(seqgendiff)
 library(compcodeR)
+```
 
-generateSequentialCountMatrix <- function(numHyp, varNum, diffExp){
+```{r simulate_data_fct}
+generateSequentialCountMatrix <- function(numHyp, varNum, diffExp, nSamples = 5){
```

Comment:
I added an `nSamples` parameter. General rule of thumb, if you're finding yourself repeating a parameter value within a function, parameterize it!